tidb-cloud/backup-and-restore.md

```diff
@@ -41,6 +41,20 @@ By the automatic backup, you can back up the cluster data every day at the backu
 
     - In **Backup Retention**, configure the minimum backup data retention period.
 
+    - Turn on or off the **EBS snapshot** feature.
+
```

Comment:
Question: Do we need to mention this feature is in beta and not recommended for production environments?

Reply:
I do not think so.